src/views/BondV2/bond.scss

```diff
@@ -144,3 +144,10 @@
 .web3modal-modal-lightbox {
   z-index: 70 !important;
 }
+
+.bond_discount_positive {
```

Comment:
We might want to check with design on the correct shades of green/red

Reply:
Definetly.  Who is design?